server/src/main/java/io/druid/server/QueryResource.java

```diff
@@ -107,9 +107,9 @@ public QueryResource(
   @Produces(MediaType.APPLICATION_JSON)
   public Response getServer(@PathParam("id") String queryId)
   {
+    log.info("Get cancel query for queryId[%s]", queryId);
```

Comment:
Debug level please.

Also suggest "Received cancel request for query [%s]"  or similar